packages/valory/customs/dynamic_betting_strategy/dynamic_betting_strategy.py

```
@@ -0,0 +1,59 @@
+
+
+def dynamic_bet_amount(
+    confidence: float,
+    bet_amount_per_threshold: Dict[str, int],
+    market_trend: float,
+    last_bet_outcome: str
+) -> Dict[str, Union[int, Tuple[str]]]:
+    """Adjust the bet amount based on confidence and external factors."""
+    
+    threshold = str(round(confidence, 1))
+    base_bet_amount = bet_amount_per_threshold.get(threshold, None)
```

Ruff

Use `bet_amount_per_threshold.get(threshold)` instead of `bet_amount_per_threshold.get(threshold, None)`

Replace `bet_amount_per_threshold.get(threshold, None)` with `bet_amount_per_threshold.get(threshold)`

(SIM910)